python/lsst/sconsUtils/builders.py

```diff
@@ -602,20 +602,22 @@ def makeVersionModule(target, source, env):
 
             what = "__dependency_versions__"
             names.append(what)
-            outFile.write("%s = {\n" % (what))
-            for name, mod in env.dependencies.packages.items():
-                if mod is None:
-                    outFile.write("    '%s': None,\n" % name)
-                elif hasattr(mod.config, "version"):
-                    outFile.write("    '%s': '%s',\n" % (name, mod.config.version))
-                else:
-                    outFile.write("    '%s': 'unknown',\n" % name)
+            outFile.write("%s = {" % (what))
+            if len(env.dependencies.packages.items()) != 0:
```

Comment:
Is the .items and != 0 needed here? Would

```python
if env.dependencies.packages:
```
also work?

Reply:
Seems to work, thanks!